pkg/api/serve.go

```diff
@@ -85,7 +85,7 @@ func Serve(
 		actions,
 		auditChecker,
 		logger,
-		emailer,
+		*emailer,
```

Comment:
You want to keep a pointer to emailer - later when you add the tests you will need to replace it with interface

Reply:
Done

Reply:
The previous feedback I wrote - passing emailer implementation as a pointer and not copy the value from the creation, is true here too.
Don't want to copy the emailer struct by value - we like to pass the pointer we created also the controller.

Reply:
Got it, made two pushes one implementing it via pointers (i.e. the emailer is passed by reference) and the second via interface. I think the interface one is more proper.